font-src/glyphs/letter/latin/lower-n.ptl

```diff
@@ -174,16 +181,16 @@ glyph-block Letter-Latin-Lower-N : begin
 
 		create-glyph "RInsular.\(suffix)" : glyph-proc
 			include : MarkSet.capDesc
-			include : Body CAP SB RightSB (0 + TailY - HalfStroke) Stroke
-			include : RetroflexHook.rExt RightSB (0 + TailY - HalfStroke)
+			include : Body CAP SB RightSB (XH - SmallArchDepthB + O) Stroke
```

Comment:
We *should* use the full `RetroflexHook` for tails that carry semantics.
Just change the height used here to `(0 - descender - HalfStroke)`.